staging_service/app.py

```diff
@@ -108,31 +104,45 @@ def _file_type_resolver(path: PathPy) -> FileTypeResolution:
         return FileTypeResolution(unsupported_type=ext)
 
 
+def _make_dts_file_resolver() -> Callable[[Path], FileTypeResolution]:
+    """Makes a DTS file resolver."""
+
+    def dts_file_resolver(_: PathPy):
+        return FileTypeResolution(parser=parse_dts_manifest)
+
+    return dts_file_resolver
+
+
 @routes.get("/bulk_specification/{query:.*}")
 async def bulk_specification(request: web.Request) -> web.json_response:
     """
     Takes a `files` query parameter with a list of comma separated import specification file paths.
     Returns the contents of those files parsed into a list of dictionaries, mapped from the data
     type, in the `types` key.
 
-    :param request: contains a comma separated list of files, e.g. folder1/file1.txt,file2.txt
-
-    TODO: since JSON files are rather generic and we might want to use a different JSON bulk-spec
-    format later, add a separate query parameter to request that the selected file is treated as a
-    Data Transfer Service manifest.
+    :param request: contains the URL parameters for the request. Expected to have the following:
+        * files (required) - a comma separated list of files, e.g. folder1/file1.txt,file2.txt
+        * dts (optional) - if present, and has the value "1", this will treat all of the given
+          files as DTS manifest files, and attempt to parse them accordingly.
     """
     username = await authorize_request(request)
-    files = parse_qs(request.query_string).get("files", [])
+    params = parse_qs(request.query_string)
+    files = params.get("files", [])
     files = files[0].split(",") if files else []
     files = [f.strip() for f in files if f.strip()]
     paths = {}
     for f in files:
         p = Path.validate_path(username, f)
         paths[PathPy(p.full_path)] = PathPy(p.user_path)
+    as_dts = params.get("dts", ["0"])[0] == "1"
+
     # list(dict) returns a list of the dict keys in insertion order (py3.7+)
+    file_type_resolver = _file_type_resolver
+    if as_dts:
+        file_type_resolver = _make_dts_file_resolver()
```

Comment:
This is the core of the change. If the URL has `dts=1` in it, then treat all files as DTS manifests and parse them that way.

Doing this avoids either code duplication in the `file_parser` module, and avoids opening the door to annoying branching logic if we want to add other JSON formats.